tools/cldr-code/src/main/java/org/unicode/cldr/util/PathHeader.java

```diff
@@ -574,42 +582,6 @@ public int compareTo(PathHeader other) {
         }
     }
 
-    public int compareHeader(PathHeader other) {
```

Comment:
Add a comment that if we ever have to compare just the header or just the code, the code to do that is in release 44. (These are a pain to get right.)

Reply:
I've just added such a comment in the method compareTo